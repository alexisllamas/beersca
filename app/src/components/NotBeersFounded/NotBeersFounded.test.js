import React from 'react';
import mount from 'enzyme/mount';
import NotBeers from '.';

describe('NotBeers component', () => {
  it('should render correctly', () => {
    const wrapper = mount(<NotBeers />);
    expect(wrapper).toMatchSnapshot();
  });
});
