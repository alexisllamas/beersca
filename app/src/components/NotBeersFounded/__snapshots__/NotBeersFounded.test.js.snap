// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotBeers component should render correctly 1`] = `
<NotBeersFounded>
  <div
    className="notification is-danger"
  >
    Sorry, we found no beers :(
  </div>
</NotBeersFounded>
`;
